src/utils/config.ts

```diff
@@ -23,6 +23,12 @@ const configParsers = {
 
 		return key;
 	},
+	locale(key: string) {
+		parseAssert('locale', key, 'Cannot be empty');
+		parseAssert('locale', /^[a-z-]+$/i.test(key), 'Must be a valid locale (letters and dashes/underscores). You can consult the list of codes in: https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes');
```

Comment:
```suggestion
		parseAssert('locale', /^[a-z-]+$/i.test(key), 'Must be a valid locale (letters and dashes/underscores). You can consult the list of codes in: https://wikipedia.org/wiki/List_of_ISO_639-1_codes');
```

Removing `en` so it gets localized to each viewer.